### DISCUSSION TITLE
Links open in a Frame

### DISCUSSION THREAD

**Discussion by anrilweb** [YOURLS/YOURLS#3701] [Q&A] [answered]
The links open in a Frame, and don't redirect to the correct page,

**Comment by dgw** [comment, marked as answer]
Not a single piece of information that would help diagnose this. No YOURLS version, no screenshot, nothing.

Random guess: Did you maybe enable the toolbar plugin that ships with YOURLS? It looks like this in the plugin list (but mine is deactivated):

<img width="718" alt="image" src="https://github.com/YOURLS/YOURLS/assets/164140/513dbfc2-2c11-4cc9-ae1b-bd4031f52b5e">

**Reply by anrilweb** [reply]
Hi, thank you for the reply, yes inattention, there is such a thing..... You are right, the problem was a plugin enabled.
Здраствуйте, благодарю за ответ, да невнимательность, есть такое... Вы правы, проблемой был включеный плагин. (Ссылки открываются во фрейме и не перенаправляют на нужную страницу)
